Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.12.3
+
+WORKDIR /workspace
+
+COPY requirements.txt .
+RUN python -m pip install --upgrade --no-cache-dir pip setuptools
+RUN python -m pip install --upgrade --no-cache-dir -r requirements.txt
```

Comment:
I suggest adding the EOL at the end of the last line, avoiding this display of a line without EOL.

Reply:
Updated